packages/collector/Collector_Zip.php

```diff
@@ -0,0 +1,44 @@
+<?php
+use ZipStream\ZipStream;
+use ZipStream\Option\Archive;
+
+
+function collector_zip_wp_content($zip)
+{
+	$callback = function ($realPath, $packPath) use ($zip, &$callback) {
+		if(is_file($realPath))
+		{
+			$zip->addFileFromPath($packPath, $realPath);
+		}
+		else if(is_dir($realPath))
+		{
+			collector_iterate_directory($realPath, ABSPATH, $callback);
+		}
+	};
+
+	collector_iterate_directory(ABSPATH . '/wp-content/', ABSPATH, $callback);
+}
+
+function collector_open_zip()
+{
+	$options = new Archive();
+	$options->setSendHttpHeaders(true);
+	$zip = new ZipStream(
+		collector_get_tmpfile('collector', 'zip'),
```

Comment:
`collector_get_tmpfile` is only used here. What do you think about inlining it?

```suggestion
		'collector-package-'. date('Y-m-d_H-i-s') . '.zip',
```

or even just

```suggestion
		'export.zip',
```